wee_slack.py

```diff
@@ -30,9 +31,15 @@
 # hack to make tests possible.. better way?
 try:
     import weechat
-except:
+except ImportError:
     pass
 
+try:
+    basestring     # Python 2
+    unicode
+except NameError:  # Python 3
+    basestring = unicode = str
```

Comment:
I didn't say using a tuple was necessarily wrong, but I wondered the reason for using it. :) Or wasn't there any specific reason?

Reply:
I originally thought there was a reason for making it a tuple of one but further test proved that I was wrong.

Reply:
Ah, okay :)